### DISCUSSION TITLE
Failed to simulate opensea transaction with hardhat node fork mode

### DISCUSSION THREAD

**Discussion by wong2** [NomicFoundation/hardhat#3185] [Troubleshooting]
Hi, I'm running a hardhat node with fork mode (fork from mainnet). Then I send an OpenSea buying order transaction to the node, but the transaction reverted without reason. 

Do you have any idea?

**Comment by fvictorio** [comment]
Hopefully the stack trace of the reverted transaction should give you a hint of what happened. At the very least, you should get the address of the contract that reverted, and you should be able to start investigating from there.

Other than that, it's hard to help without more information.
